bwapi/bwresources.py

```diff
@@ -598,7 +643,7 @@ def _name_to_id(self, attribute, setting):
                 setting = [setting]
             ids = []
             for s in setting:
-                ids.append(self.tags.ids[s])
+                ids.append(self.tags.get_resource_id(s))
```

Comment:
There are a couple more of tags.ids / authorlists.ids / locationlists.ids in this file as the _name_to_id and _id_to_name methods are implemented multiple times. Can we confirm that these 2 methods still work after switching to the new approach? Thanks!

Reply:
Thanks for raising - these seem to have slipped through the net - I've fixed now